.gitignore

```diff
@@ -1,12 +1,28 @@
+# dependencies
+website2.0/node_modules
```

Comment:
Hi, could we omit the wesite2.0 directory? Set all needed files under the root directory directly is fine.

Reply:
Yes Juzhiyuan .It would be better to under have the root directory   .